src/test/java/com/emc/mongoose/storage/driver/kafka/integration/ReadRecordTest.java

```diff
@@ -0,0 +1,65 @@
+package com.emc.mongoose.storage.driver.kafka.integration;
+
+import com.emc.mongoose.storage.driver.kafka.util.docker.KafkaNodeContainer;
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.HashMap;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.MockConsumer;
+import org.apache.kafka.clients.consumer.OffsetResetStrategy;
+import org.apache.kafka.common.TopicPartition;
+import org.junit.*;
+
+public class ReadRecordTest {
+  private static KafkaNodeContainer kafkaNodeContainer;
+  private MockConsumer<String, String> mockConsumer;
+  private static final String TOPIC_NAME = "test-topic";
```

Comment:
please use some test local topic name, e.g. "topic" + ReadRecordTest.class.getSimpleName()
different tests may use same kafka instance in future
so we need to avoid the collision